ExotischNederland/Menus/GameMenu.cs

```diff
@@ -0,0 +1,211 @@
+            this.authenticatedUser = _authenticatedUser;
+        }
+
+        public Dictionary<string, string> GetMenuItems()
+        {
+            Dictionary<string, string> menuItems = new Dictionary<string, string>();
+
+            // Add game management options if the user has the "Beheerder" role
+            if (authenticatedUser.Permission.CanManageGames())
+            {
+                menuItems.Add("createGame", "Create a new game");
+                menuItems.Add("editGame", "Edit an existing game");
+                menuItems.Add("deleteGame", "Delete a game");
+            }
+
+            // Add game-playing option for "Familie" and "Kinderen" roles
+            if (authenticatedUser.Permission.CanPlayGames())
+            {
+                menuItems.Add("playGame", "Play a game");
+            }
+
+            menuItems.Add("back", "Return to main menu");
+            return menuItems;
+        }
+
+        public void Show()
+        {
+            menuItems = this.GetMenuItems();
+            while (true)
+            {
+                string selected = Helpers.MenuSelect(this.menuItems, true);
+
+                if (selected == "createGame")
+                {
+                    CreateGame();
+                }
+                else if (selected == "editGame")
+                {
+                    EditGame();
+                }
+                else if (selected == "deleteGame")
+                {
+                    DeleteGame();
+                }
+                else if (selected == "playGame")
+                {
+                    PlayGame();
+                }
+                else if (selected == "back")
+                {
+                    break;
+                }
+            }
+        }
+
+        private void CreateGame()
+        {
+            Console.Clear();
+            Console.WriteLine("Create a New Game");
+            Console.Write("Enter the title of the game: ");
+            string title = Console.ReadLine();
+            Console.Write("Enter the description of the game: ");
+            string description = Console.ReadLine();
+
+            Console.Write("Enter route ID for the game: ");
+            int routeId;
+            if (int.TryParse(Console.ReadLine(), out routeId))
+            {
+                Route route = Route.Find(routeId);
+                if (route != null)
+                {
+                    Game game = Game.Create(route, title, description);
+                    Console.WriteLine($"Game '{title}' created successfully.");
+                }
+                else
+                {
+                    Console.WriteLine("Invalid route ID. Game creation failed.");
+                }
+            }
+            else
+            {
+                Console.WriteLine("Invalid route ID input.");
+            }
+
+            Console.WriteLine("Press any key to return to the menu.");
+            Console.ReadKey();
+        }
+
+        private void EditGame()
```

Comment:
The Form class should be used here, validation is nice, but to be able to leave a field empty to keep the current value there is additional logic needed. This is already included in the Form class